src/components/Registration/RegistrationStep1.tsx

```diff
@@ -0,0 +1,349 @@
+import React from 'react';
+import CheckoutStep from './CheckoutStep';
+import Heading from './Heading';
+import styled from '@emotion/styled';
+import { DatePicker, LocalizationProvider } from '@mui/lab';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import {
+  Box,
+  FormControl,
+  MenuItem,
+  Select,
+  Typography,
+  TextField,
+  List,
+  ListItem,
+  ListItemText,
+  Button
+} from '@mui/material';
+import { ArrowBack, ArrowForward } from '@mui/icons-material';
+import { useNavigate } from 'react-router-dom';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
+import { Controller, useForm } from 'react-hook-form';
+import FormHelperText from '@mui/material/FormHelperText';
+interface RegistrationInfo {
```

Comment:
```suggestion
import FormHelperText from '@mui/material/FormHelperText';

interface RegistrationInfo {
```

phần import luôn luôn có 1 new line với code ở dưới